README.md

```
@@ -823,7 +823,7 @@ Layer B makes sense when you specifically want the premium model's **thinking an
 | HTML | meta, JSON-LD, data-ai* | Strip tags/attrs |
 | Markdown | YAML frontmatter AI keys | Drop keys + Layer A body |
 | MP4 / MOV / M4A / M4V | ISOBMFF `jumb`/`uuid` boxes (same mechanism as AVIF/HEIC) + `moov/udta` generator tags | Drop boxes |
-| WAV | RIFF `LIST INFO` chunk, embedded `id3 ` chunk | Drop chunks |
+| WAV | RIFF `C2PA` / `LIST INFO` chunks, embedded `id3 ` chunk | Drop chunks |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)